manifests/init.pp

```diff
@@ -130,7 +114,7 @@
 class simp (
   Enum['simp','simp_lite','poss'] $scenario                   = 'simp',
   Boolean                         $enable_data_includes       = true,
```

Comment:
Based on the change to the logic below, `$enable_data_includes` should be removed.